site/content/en/blog/releases/0.12.0-rc.md

```diff
@@ -14,9 +14,7 @@ We've also worked to complete functionality across GameServer SDKs, but we still
 
 We are on track for a 1.0 release in September, with only a [few features left to implement](https://github.com/googleforgames/agones/milestone/14).
 
-Check the <a href="https://github.com/googleforgames/agones/tree/release-0.12.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
-
-**Implemented enhancements:**
+Check the <a href="https://github.com/googleforgames/agones/tree/release-0.12.0-rc" >README</a> for details on features, installation and usage.
```

Comment:
seems like this shouldn't have the rc extension in it?

Reply:
That's the release candidate post - so, yes it should? (the change here is removing the `data-proofer-ignore` tag from the link)